vtds_cluster_kvm/private/scripts/deploy_cluster_to_blade.py

```diff
@@ -1868,8 +804,8 @@ def main(argv):
     # the blades).
     nodes = [
         VirtualNode(
+            config,
             node_class,
```

Comment:
REVIEWERS: node_connected_networks() is now computed in the VirtualNode or its Node Builder, so it does not need to be passed to the constructor anymore.